### ISSUE TITLE
Template parse errors in RC6

### ISSUE THREAD

**Issue by aramin** [500tech/angular2-tree-component#40]
With RC6  all components and pipes must now be declared via an NgModule. Unfortunately the following error message pops up since updating:

```
Can't bind to 'node' since it isn't a known property of 'TreeNode'. ("
      <TreeNode
        *ngFor="let node of treeModel.roots"
        [ERROR ->][node]="node">
      </TreeNode>
    </div>
"): TreeComponent@4:8
```

 Any clue on this?

**Comment by Gillardo** [comment]
I am getting this error as well....has this been done now?

**Comment by adamkleingit** [comment]
I hope to handle RC6 until this weekend

-- 
- Adam Klein <http://il.linkedin.com/in/adamklein500>
- CTO @ 500Tech.com <http://500tech.com/>
- AngularJS, ReactJS, NodeJS
- Development, consulting, training and public speaking
- (+972) 52 747 5633

**Comment by isaacplmann** [comment]
Proposed update in #43

**Comment by adamkleingit** [comment]
Fixed in #43
